.github/workflows/conductor.yml

```diff
@@ -0,0 +1,31 @@
+on:
+  repository_dispatch:
+    types:
+      - dependency_update
+
+name: Private Packagist Conductor
+
+permissions:
+  contents: write
+
+jobs:
+  conductor:
+    name: Private Packagist Conductor
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          sparse-checkout: |
+            composer.json
+            composer.lock
+      - name: Get desired PHP version
+        id: supported-versions-matrix
+        uses: WyriHaximus/github-action-composer-php-versions-in-range@v1
+      - name: Install PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: "${{ steps.supported-versions-matrix.outputs.lowest }}"
+          coverage: "none"
+          extensions: ${{ join(fromJson(steps.supported-versions-matrix.outputs.extensions), ',') }}
```

Comment:
@naderman Some quality of life improvements that ensures we have all the extensions we need installed, plus run on the lowest version targetted in `composer.json`.